### ISSUE TITLE
The controller's interface resides in a different sub-module, and its comments are not being recognized.

### ISSUE THREAD

**Issue by yumiaoxia** [TongchengOpenSource/smart-doc#545]
## Your Environment(您的使用环境)
spring boot:2.4.5
![Uploading WX20230712-024025@2x.png…]()

As mentioned, the `TestApi` is located in a dependent sub-module, and its comments are not being recognized.
如上，TestApi在依赖的子模块， 无法将其注释识别

**Comment by yumiaoxia** [comment]
```java
@RestController
public class TestController implements TestApi {

    @Override
    public TestVO test() {
        return null;
    }
}

``` 
The `TestController` is situated within the starter module.
TestController 在starter模块中


```java
/**
 * 测试相关
 */
@RequestMapping("api/test")
public interface TestApi {

    /**
     * 测试
     * @return {@link TestVO}
     */
    @GetMapping
    TestVO test();
}

``` 
The `TestApi` is housed in the api module, which is a dependency of the starter module. However, smart-doc is unable to recognize the annotations on the `TestApi` interfaces.

**Comment by shalousun** [comment]
This is not considered a bug. Firstly, it is recommended to upgrade the tool. Secondly, you may refer to the troubleshooting section in the documentation. Comments need to be accessible to smart-doc for parsing. Please check the following link for more information: [https://smart-doc-group.github.io/#/zh-cn/faq](https://smart-doc-group.github.io/#/zh-cn/faq)

**Comment by abing22333** [comment]
This issue has been fixed in version 3.0.0.   
 #597
